pkg/cloud/cloud.go

```diff
@@ -77,6 +78,10 @@ func NewProviderFactory(restMapper meta.RESTMapper, kubeClient kubernetes.Interf
 }
 
 func (f *providerFactory) Get(config *configv1alpha1.SubmarinerConfig, eventsRecorder events.Recorder) (Provider, bool, error) {
+	if config.Annotations["submariner.io/skip-cloud-prepare"] == strconv.FormatBool(true) {
```

Comment:
```suggestion
	if strconv.ParseBool(config.Annotations["submariner.io/skip-cloud-prepare"]) {
```

Reply:
Can't do exactly like that since `ParseBool` also returns an error if it's not "true" or "false" or allowed synonyms or if it's empty (ie annotation not present).  I think it's easiest to just accept "true" and anything else is false. This is just an undocumented backdoor.

Reply:
Ah yes, it returns `(bool, error)`.